etna/pipeline/base.py

```diff
@@ -331,7 +354,25 @@ def _add_forecast_borders(
             - ts[backtest_forecasts.index.min() : backtest_forecasts.index.max(), :, "target"]
         )
 
-        sigma = np.std(residuals.values, axis=0)
+        len_backtest, num_segments = residuals.shape
```

Comment:
May be it is better to make a separate method for this check?

Reply:
Ok, will be done.